.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+---
+version: 2.1
+
+docker_auth: &docker_auth
+  username: $DOCKERHUB_USERNAME
+  password: $DOCKERHUB_PASSWORD
+
+defaults: &defaults
+  docker:
+    - image: greenpeaceinternational/circleci-base:latest
+      auth:
+        <<: *docker_auth
+  working_directory: /home/circleci/app
+
+jobs:
+  build-tokens:
+    <<: *defaults
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - checkout
+      - run:
+          name: Convert JSON tokens into a SASS file
+          command: |
+            chmod +x ./bin/build.py
```

Comment:
You can do that locally and re-commit the scripts. Executable mode is preserverd by git.